content/00.front-matter.md

```diff
@@ -12,7 +12,7 @@ _A DOI-citable version of this manuscript is available at <https://doi.org/DOI_H
 <small><em>
 This manuscript
 {% if ci_source is defined -%}
-([permalink](https://{{ci_source.repo_owner}}.github.io/{{ci_source.repo_name}}/v/{{ci_source.commit}}/))
+([permalink]({{ci_source.manuscript_url}}))
 {% endif -%}
 was automatically generated
 {% if ci_source is defined -%}
```

Comment:
It's not obvious to me which commit we should show in line 19 below for AppVeyor builds.  The manuscript filename contains the triggering commit, but this link contains the merge commit for a pull request build.  However, the triggering commit doesn't correspond to the `ci_source.repo_slug` tree.  It resides in the tree of the forked repository.

I don't see a better option that what is already here.  I do expect the difference between the filename and link could cause confusion.

Reply:
Yes it is confusing, but I think it's important that the manuscript contains the commit that actually created it.